.drone.yml

```diff
@@ -30,6 +30,7 @@ services:
     image: postgres:9.5
     environment:
       POSTGRES_DB: test
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
This is required due to docker-library/postgres#658

Reply:
It seems better to explicitly set POSTGRES_PASSWORD

Reply:
```suggestion
       POSTGRES_PASSWORD: postgres
```